### PR TITLE
Fix Zcash memo handling

### PR DESCRIPTION
**Pull request by peachbits**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205041005078327

### CODE REVIEW

**Comment by peachbits** [comment]
Draft until https://github.com/EdgeApp/react-native-zcash/pull/38 is published

File: src/zcash/ZcashEngine.ts [hunk 1]
```diff
@@ -261,20 +261,19 @@ export class ZcashEngine extends CurrencyEngine<
...
     }
 
-    const memos: EdgeMemo[] = []
-    if (tx.memo != null) {
-      memos.push({
+    const edgeMemos: EdgeMemo[] = tx.memos
+      .filter(text => text !== '')
+      .map(text => ({
         type: 'text',
-        value: tx.memo
```
Comment (swansontec) [inline src/zcash/ZcashEngine.ts:268]: The default is `memoName: 'memo'`, which is what we want to show in the GUI. However, we could add this field to be explicit about what this thing is called.

**Review by swansontec — APPROVED** [review]

File: src/zcash/ZcashEngine.ts (hunk 1 above)
Comment (peachbits) [reply src/zcash/ZcashEngine.ts:268]: Zcash calls them memo so the default is fine

Comment (peachbits) [reply src/zcash/ZcashEngine.ts:268]: changed it anyway

**Pull request merged**